### PR TITLE
Pull Request #1: Initialize Dependences & Create Structure

### PR DESCRIPTION
**Pull request by martinsung00**
Added dependencies, configuration files, and created a simple structure.

### CODE REVIEW

**Comment by hugoxiong5** [comment]
@martinsung00 Overall, looks pretty good to me, but I would suggest updating the package.json to reflect your authorship. Right now the name and author, etc. are still copy pasted over from the original repo.

Also, why does the webpack config file have so many eslint ignore lines above it? I don't think I needed to do that for my webpack.

**Comment by martinsung00** [comment]
> @martinsung00 Overall, looks pretty good to me, but I would suggest updating the package.json to reflect your authorship. Right now the name and author, etc. are still copy pasted over from the original repo.
> 
> Also, why does the webpack config file have so many eslint ignore lines above it? I don't think I needed to do that for my webpack.

Fixed the mentioned issues! Thanks for the review.

File: database-mysql/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+const connection = mysql.createConnection({
+  host     : 'localhost',
+  user     : 'root',
+  password : '',
+  database : 'proxy_primary'
+});
+
+const selectAll = function(callback) {
+  connection.query('SELECT * FROM items', function(err, results, fields) {
+    if(err) {
```
Comment (hugoxiong5) [inline database-mysql/index.js:12]: Hi

Comment (hugoxiong5) [reply database-mysql/index.js:12]: yo yo yo

**Review by hugoxiong5 — COMMENTED** [review]
blah blah overall

**Pull request merged**